Gemfile

```diff
@@ -8,6 +8,7 @@ gem 'rexml'
 gem 'hearth',
     git: 'https://github.com/smithy-lang/smithy-ruby',
     glob: 'hearth/hearth.gemspec'
+    # path: '../smithy-ruby/hearth'
```

Comment:
Should probably remove - Instead of doing this, you usually should use bundle config set local.  I think its documented in our Readme?